.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+---
+image:
+  file: .gitpod.dockerfile
+
+tasks:
+  - init: |
+      echo "Welcome to TheAlgorithms/Lua"
```

Comment:
Why is this useful? It seems to just lead to confusing spam in the console:

![Screenshot from 2023-11-08 13-33-26](https://github.com/TheAlgorithms/Lua/assets/34514239/38054d67-a039-4479-8004-abafeaa70d60)

Reply:
Removed in 1d2e149e0c79b2847f0058f465e58eb9a7562b5e.